### PR TITLE
Implement responding with method not allowed when appropiate

### PR DESCRIPTION
**Pull request by e0ff**

### CODE REVIEW

File: src/fallback_html/405.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Verse: Method Not Allowed</title>
```
Comment (GrayHatter) [inline src/fallback_html/405.html]: Let's add 405 to the title too, that way at least one of the titles has the "correct" name

File: src/fallback_html/405.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      }
+      body {
+        width: 35em;
+        margin: 0 auto;
+        font-family: Tahoma, Verdana, Arial, sans-serif;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>405: Invalid Communication</h1>
```
Comment (GrayHatter) [inline src/fallback_html/405.html]: `405: Respectful Communications`

**Review by GrayHatter — APPROVED** [review]

File: src/fallback_html/405.html (hunk 2 above)
Comment (e0ff) [reply src/fallback_html/405.html]: Done

File: src/fallback_html/405.html (hunk 1 above)
Comment (e0ff) [reply src/fallback_html/405.html]: Done

**Review by GrayHatter — APPROVED** [review]

**Pull request merged**